tests/dummy/app/pods/index/template.hbs

```diff
@@ -0,0 +1,23 @@
+<DocsHeader />
+
+<DocsHero
+  @heading = "Yadda Opinionated"
+  @byline  = "A radical take on Cucumber to solve you test maintenance woes"
```

Comment:
Is that correct? Or should it be "your"?

Reply:
Fixed.